packages/grafana-ui/src/utils/standardEditors.tsx

```diff
@@ -30,7 +30,7 @@ import { StatsPickerEditor } from '../components/OptionsUI/stats';
  * Returns collection of common field config properties definitions
  */
 export const getStandardFieldConfigs = () => {
```

Comment:
if we are calling it "commonFieldConfigs" on the panel, we should match that name here.  Either rename this to "common" or the other to "standard"

I'm not sure which is better :)

Reply:
your completely right. I think I'll rename common to standard

Reply:
yea, was thinking the same thing while reading this that we had 2 names